Dockerfile

```diff
@@ -0,0 +1,151 @@
+####################################################################################################
+# Builder image
+####################################################################################################
+FROM golang:1.13.4 as builder
+
+ARG IMAGE_OS=linux
+ARG IMAGE_ARCH=amd64
+
+RUN apt-get update && apt-get --no-install-recommends install -y \
+    git \
+    make \
+    apt-utils \
+    apt-transport-https \
+    ca-certificates \
+    wget \
+    gcc \
+    zip && \
+    apt-get clean \
+    && rm -rf \
+        /var/lib/apt/lists/* \
+        /tmp/* \
+        /var/tmp/* \
+        /usr/share/man \
+        /usr/share/doc \
+        /usr/share/doc-base
+
+WORKDIR /tmp
+
+# Install docker
+ENV DOCKER_CHANNEL stable
+ENV DOCKER_VERSION 18.09.1
+
+RUN if [ "${IMAGE_OS}" = "linux" -a "${IMAGE_ARCH}" = "amd64" ]; then \
+    	wget -O docker.tgz https://download.docker.com/linux/static/${DOCKER_CHANNEL}/x86_64/docker-${DOCKER_VERSION}.tgz; \
+    elif [ "${IMAGE_OS}" = "linux" -a "${IMAGE_ARCH}" = "arm64" ]; then \
+	wget -O docker.tgz https://download.docker.com/linux/static/${DOCKER_CHANNEL}/aarch64/docker-${DOCKER_VERSION}.tgz; \
+    fi && \
+    tar --extract --file docker.tgz --strip-components 1 --directory /usr/local/bin/ && \
+    rm docker.tgz
+
+####################################################################################################
+# Build base
+####################################################################################################
+FROM builder as argo-events-build-base
+
+# Perform the build
+WORKDIR /argo-events
+COPY . .
+# check we can use Git
+RUN git rev-parse HEAD
+
+####################################################################################################
+# Build eventbus-controller binary
+####################################################################################################
+FROM argo-events-build-base as eventbus-controller-build
+
+# eventbus-controller image
+RUN make eventbus-controller-linux
```

Comment:
We shouldn't build binaries. Binaries are built by the GitHub pipeline. The Dockerfile should just copy binaries from /dist folder to /bin